### ISSUE TITLE
Providing scss files for better modularity

### ISSUE THREAD

**Issue by andreasisaak** [noelboss/featherlight#198]
Would it be possible that you providing scss files with the package? If desired, i write the pull request :)

**Comment by marcandre** [comment]
That could be nice, except it is not trivial to do.

We currently use the `src/` files directly when developing & testing. There would have to be a "compiling process". It wouldn't be worth the annoyance just for `scss` though. But If we are to do this, we could as well move to ES6. I'd like to do that one day (like I [did with Parsley](https://github.com/guillaumepotier/Parsley.js/pull/968)). It would be easy to add `scss` processing to the gulp file.

In short: producing the scss file is easy, but it would have to be after a move to ES6. Adapting the project to ES6 is a bit more involved...

**Comment by andreasisaak** [comment]
I think you're right but there are just 3 CSS changes last year. Theoretically, we could first offer SCSS and CSS simultaneously

**Comment by Jakobud** [comment]
I have a question. Why would converting the CSS to SCSS require converting the JS to ES6 syntax first? It would be super simple to add the SCSS file and then update the `gulpfile` to add that compiling step in...

**Comment by marcandre** [comment]
You're right, we could do those in any order. I'd accept a PR for grunt to gulp and/or a PR from css to scss
